recipe/migrations/osx_arm64.txt

```diff
@@ -793,6 +793,7 @@ py-cpuinfo
 py-openimageio
 py-plumed
 py-spy
+py_rattler
```

Comment:
```suggestion
py-rattler
```

Reply:
I am not sure wether the feedstock name or the output name matters. The arch'd package is `py_rattler` if I looked correctly.